examples/projects/project.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: projects.argocd.crossplane.io/v1alpha1
+kind: Project
+metadata:
+  name: example-project.git
+spec:
+  forProvider:
+    description: "Example project"
```

Comment:
I believe it's common to only have mandatory fields in the example (e.g. the minimum necessary to create a `project`) and leave the rest to http://doc.crds.dev/github.com/crossplane-contrib/provider-argocd (this link will only work after we do a first release)